src/libs/ReportActionsUtils.js

```diff
@@ -156,18 +157,15 @@ function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
 }
 
 /**
- * This method returns the report actions that are ready for display in the ReportActionsView.
- * The report actions need to be sorted by created timestamp first, and reportActionID second
- * to ensure they will always be displayed in the same order (in case multiple actions have the same timestamp).
- * This is all handled with getSortedReportActions() which is used by several other methods to keep the code DRY.
+ * A helper method to filter out report actions keyed by sequenceNumbers.
  *
  * @param {Object} reportActions
  * @returns {Array}
  */
-function getSortedReportActionsForDisplay(reportActions) {
+function filterOutDeprecatedReportActions(reportActions) {
```

Comment:
👍 nice I like that you refactored this into a separate function and gave it a descriptive name.